uber/custom_tags.py

```diff
@@ -1,55 +1,107 @@
 from uber.common import *
 
 
-@register.filter
-def datetime(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
-    return ' '.join(dt.astimezone(c.EVENT_TIMEZONE).strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm')
+def safe_string(str):
+    return JinjaEnv.env().filters['safe'](str)
 
-from datetime import datetime  # noqa: now that we've registered our filter, re-import the "datetime" class to avoid conflicts
 
+@JinjaEnv.jinja_filter(name='date')
+def date_filter(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
+    # TODO: if this is a datetime.time object, do we have to correct for timezone with .astimezone()?
+    return ' '.join(dt.strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm') if dt else ''
 
-@register.filter
+
+@JinjaEnv.jinja_filter(name='datetime')
+def datetime_filter(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
+    return ' '.join(dt.astimezone(c.EVENT_TIMEZONE).strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm') if dt else ''
+
+
+@JinjaEnv.jinja_filter()
+def yesno(value, arg=None):
```

Comment:
Was this pulled in from somewhere, or did you write it from scratch?  I'm only asking because if we copied this from some other repo, we should probably give attribution.  (Might not be strictly required if e.g. the other project has a compatible FOSS license, just a nice thing to do.)